include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -348,25 +400,50 @@ struct ModelMetadata : Base<OrtModelMetadata> {
   int64_t GetVersion() const;
 };
 
+/** \brief Wrapper around ::OrtSession
+*
+*/
 struct Session : Base<OrtSession> {
-  explicit Session(std::nullptr_t) {}
+  explicit Session(std::nullptr_t) {} ///< \brief Create an empty Session object, must be assigned a valid one to be used 
   Session(Env& env, const ORTCHAR_T* model_path, const SessionOptions& options);
   Session(Env& env, const ORTCHAR_T* model_path, const SessionOptions& options, OrtPrepackedWeightsContainer* prepacked_weights_container);
   Session(Env& env, const void* model_data, size_t model_data_length, const SessionOptions& options);
 
-  // Run that will allocate the output values
+  /** \brief Run the model returning results in an Ort allocated vector.
```

Comment:
>Ort allocated vector [](http://example.com/codeflow?start=51&length=20)

I think it could be clearer about ORT allocating the output values (something like the old comment)

Reply:
What else would 'Ort allocated' mean?

Reply:
it says "Ort allocated vector", which sounds like a vector that ORT allocates